app/javascript/src/components/Dashboard/Notes/MenuBar/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import { MenuBar } from "neetoui/layouts";
+import { useTranslation } from "react-i18next";
+
+import { VIEWS } from "../constants";
+
+const Menu = ({ showMenu }) => {
+  const { t } = useTranslation();
+
+  return (
+    <MenuBar showMenu={showMenu} title={t("common.notes")}>
+      {VIEWS.map(({ category, count }) => (
+        <MenuBar.Block
+          active={category === VIEWS[0].category}
+          count={count}
+          key={category}
```

Comment:
Is `category` a primitive value or a compound value? Because we are supposed to pass a primitive value like say a Number or String to `key` prop. Please read more upon the necessity of the `key` prop and how it's utilised by React. Refer: https://academy.bigbinary.com/learn-react/react-best-practices/using-unique-keys-for-list-elements

The reason why I even had to ask the above question was because of the non-semantic naming convention that you have followed here. The variable name `category` suggests in all aspects that it's an `Object`. If it was the name of the category, then it ought to have been named `categoryName`. Or if it was meant to be an object itself then pass `category.name` to `key` prop.

Reply:
Got it, my bad, will fix this convention issue.